### DISCUSSION TITLE
What is "discord.php"?

### DISCUSSION THREAD

**Discussion by iametanik** [Cvolton/GMDprivateServer#787] [Q&A]
Idk, how it works. I have a discord and i wanna to configure that. 
file code: `<?php
$discordEnabled = 0;
$secret = "please change this if you intend to use a discord bot with the server";
$bottoken = "please change this to a discord bot token if you use one";
?>`

**Comment by Cvolton** [comment]
relates to this
https://github.com/Cvolton/pythonCvoltonBot
the implementation needs a redux tbh

**Comment by ghost** [comment]
The script "discord.php" runs a Discord bot on your server for linking accounts with Discord, song and level information, etc. The purpose of it is to reply with information about your GDPS.

**Reply by ghost** [reply]
I don't fully understand it, but I looked through the script Cvolton linked. Correct me if I'm wrong.
